target/linux/ramips/image/mt7621.mk

```diff
@@ -753,6 +753,21 @@ define Device/iptime_a8004t
 endef
 TARGET_DEVICES += iptime_a8004t
 
+define Device/iptime_t5004
+  $(Device/dsa-migration)
```

Comment:
This should be dropped. Your device wasn't supported before DSA migration.

Reply:
NO. On ramips/mt7621 this is required for new devices as well, since we unconditionally set version 1.1 in board.d/05_compat-version

One could think of adding a different definition with different message here for new devices, though. (In the past, having the DSA migration message here came handy for new devices as well when people backported it to 19.07 without DSA).

Reply:
Sorry. I remember, my WAC104 device was added without it.  But I missed info about 1.1 in 05_compat-version.